snapcraft/utils.py

```diff
@@ -25,13 +27,16 @@
 from dataclasses import dataclass
 from getpass import getpass
 from pathlib import Path
-from typing import Iterable, List, Optional
+from typing import TYPE_CHECKING, Iterable, List, Optional
 
 from craft_cli import emit
 from craft_parts.sources.git_source import GitSource
 
 from snapcraft import errors
 
+if TYPE_CHECKING:
```

Comment:
leftover?

Reply:
Yep, good catch thanks!

I wonder if it'd be possible to get ruff's formatter to just remove these sorts of if statements **ONLY IF** they're checking for `TYPE_CHECKING`